test/org/sagebionetworks/bridge/services/ParticipantServiceTest.java

```diff
@@ -457,7 +464,8 @@ public void updateParticipantWithExternalIdValidationAddingId() {
         assertTrue(options.get(DATA_GROUPS).contains("group2"));
         assertTrue(options.get(LANGUAGES).contains("de"));
         assertTrue(options.get(LANGUAGES).contains("fr"));
-        assertNull(options.get(EXTERNAL_IDENTIFIER));
+        assertNull(options.get(EXTERNAL_IDENTIFIER)); // can't set this
+        assertNull(options.get(TIME_ZONE)); // can't set this
```

Comment:
Should time zone be a separate test instead of bundling it with external ID tests? Having modular tests makes things easier to test and reason about.

Also, should we have separate tests for adding time zone (no time zone to new time zone) vs changing time zone?